tf2_ros/include/tf2_ros/static_transform_broadcaster.h

```diff
@@ -54,6 +54,22 @@ class StaticTransformBroadcaster{
   TF2_ROS_PUBLIC
   StaticTransformBroadcaster(rclcpp::Node::SharedPtr node);
 
+  template<class AllocatorT = std::allocator<void>>
+  StaticTransformBroadcaster(
+    rclcpp::node_interfaces::NodeTopicsInterface::SharedPtr node_topics_interface)
+  : node_topics_interface_(node_topics_interface)
```

Comment:
I am unsure whether this class should store a copy of the node interface shared_ptr or if it's superfluous.

Reply:
If it is unused, I suppose we could drop it. I don't feel strongly about it. You don't need to keep it for any lifecycle (C++ not ROS lifecycle) reason, the publisher you create will take care of that.